### PR TITLE
Add option to slave to secure appid

### PR DESCRIPTION
**Pull request by jrschumacher**

### CODE REVIEW

File: src/lib/virtru.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const uuid = '_' + Math.random().toString(36).substr(2, 9);
```
Comment (jherwitz) [inline src/lib/virtru.js]: Can we please make this a super obvious placeholder? Like literally `PLACEHOLDER` or `REDACTED`?

File: src/lib/virtru.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+function replaceAppId(data) {
+  if (data.appId) {
+    const delegatedAppId = `<appid:${uuid()}:appid>`;
```
Comment (jherwitz) [inline src/lib/virtru.js]: Why the brackets? I don't see these in my local storage appId.

File: src/lib/virtru.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+const uuid = '_' + Math.random().toString(36).substr(2, 9);
+
+function replaceAppId(data) {
+  if (data.appId) {
```
Comment (jherwitz) [inline src/lib/virtru.js]: How can we make this as robust as possible? Can we sync up on this check offline?

File: src/lib/virtru.js (hunk 1 above)
Comment (jrschumacher) [reply src/lib/virtru.js]: For sure, but it needs to be unique. Cause we can have multiple appIds per session.

File: src/lib/virtru.js (hunk 2 above)
Comment (jrschumacher) [reply src/lib/virtru.js]: It may be too much, but I was just making something that would be unique and easy to replace.

Comment (jrschumacher) [reply src/lib/virtru.js]: Suggestions?

Comment (jherwitz) [reply src/lib/virtru.js]: Gotcha. Seems like a good idea to me.

Comment (jrschumacher) [reply src/lib/virtru.js]: NVM going to use your suggestion above.

**Pull request merged**